pkg/webservice/dao/queue_info.go

```diff
@@ -28,7 +28,7 @@ type PartitionQueueDAOInfo struct {
 	ID                     string                  `json:"id"`        // no omitempty, id should not be empty
 	QueueName              string                  `json:"queuename"` // no omitempty, queue name should not be empty
 	Status                 string                  `json:"status,omitempty"`
-	Partition              string                  `json:"partition"` // no omitempty, queue name should not be empty
+	PartitionID            string                  `json:"partition_id"` // no omitempty, partition id should not be empty
```

Comment:
I may have missed something, but it seems the `Partition` attribute is being replaced here. Shouldn't we retain the original and introduce a new `PartitionID` attribute instead? 
This approach could help avoid potential conflicts when syncing with the upstream in the future.